contacts-app/src/components/Input/Input.jsx

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import styles from "./Input.module.css";
+import PropTypes from "prop-types";
+import shortId from "shortid";
 
 class Input extends React.Component {
   render() {
     const label = this.props.label;
     const type = this.props.type;
-    const id = this.props.htmlFor;
-    // const id = this.props.htmlFor || shortId.generate();
+    const id = this.props.htmlFor || shortId.generate();
```

Comment:
this is an interesting one, perhaps watch your component in the inspector in your browser as you type into it. If im not mistaken when using the generated id every keypress you do youll see the ID update. Its not the worst, but it could slow things down if its done lots of times